client/extensions/woocommerce/state/sites/shipping-zone-locations/selectors.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import { get, isObject } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { LOADING } from 'woocommerce/state/constants';
+
+const getRawShippingZoneLocations = ( state, siteId = getSelectedSiteId( state ) ) => {
```

Comment:
Do you need the Raw here? In other places getApi... was used but I think neither is needed,